plugins/search/src/components/SearchModal/SearchModal.tsx

```diff
@@ -100,7 +100,10 @@ const useStyles = makeStyles(theme => ({
   viewResultsLink: { verticalAlign: '0.5em' },
 }));
 
-export const Modal = ({ toggleModal }: SearchModalChildrenProps) => {
+export const Modal = ({
+  toggleModal,
+  searchResultChildren,
```

Comment:
```suggestion
  resultItems,
```

Reply:
I appreciate what you are saying, but also feel a bit compelled to push back on this a bit again, I think that being clear that we are passing the children to the SearchResult component is important here. Thoughts?

Reply:
Replied in the first comment, let me know what do you think 🙂